plaidml/bridge/openvino/src/plaidml_plugin/plaidml_op_crop.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "plaidml/op/op.h"
+#include "plaidml_op_crop.hpp"
+#include "plaidml_util.hpp"
+#include <functional>
+
+void OpCrop::run(const plaidml::edsl::Tensor& I, plaidml::edsl::Tensor& O) {
```

Comment:
There should be a cleaner and more efficient EDSL implementation, but probably that's not top priority